build.rs

```diff
@@ -53,7 +53,81 @@ fn main() {
         GPRCONFIG_KB_REV,
         gprconfig_kb_path.as_path(),
     );
+    let mut envs: HashMap<String, String> = env::vars()
+        .filter(|e| !e.0.ends_with("ALIRE_PREFIX"))
+        .collect();
+    let alire_path = out_dir.join("gpr_rust_alire");
+    if !alire_path.join("alire.toml").exists()
+        && !Command::new("alr")
+            .env_clear()
+            .envs(&envs)
+            .current_dir(out_dir.to_str().unwrap())
+            .args(["--no-tty", "init", "--lib", "gpr_rust_alire"])
+            .spawn()
+            .unwrap()
+            .wait()
+            .unwrap()
+            .success()
+    {
+        panic!("failed to create alire project");
+    }
+    if !Command::new("alr")
+        .env_clear()
+        .envs(&envs)
+        .current_dir(alire_path.to_str().unwrap())
+        .args(["--no-tty", "-n", "with", "libgpr2"])
+        .spawn()
+        .unwrap()
+        .wait()
+        .unwrap()
+        .success()
+    {
+        //panic!("failed to add libgpr2");
+    }
+    if !Command::new("alr")
```

Comment:
It checks the same condition than line 74, can you join the two or is it important they fail sequentially?

Reply:
I wasn't happy with the implementation of the command handling so I refactored it.